content/blog/2025-05-07-tmil-17/index.md

```diff
@@ -0,0 +1,139 @@
++++
+title = "Linebender in May 2025"
+authors = ["Olivier Faure"]
++++
+
+Linebender is an informal open-source organization working on various projects to advance the state of the art in GUI for [the Rust programming language](https://rust-lang.org).
+
+
+## RustWeek
+
+This month saw the [RustWeek 2025](https://rustweek.org/) conference, in Utrecht in the Netherlands.
+
+A lot of things happened!
+
+### The talks
+
+Raph Levien gave a talk titled *Faster, easier 2D vector rendering*, about Vello's new architecture.
+The talk had some nice visuals and a pretty cool demo!
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/_sv8K190Zps?si=s-NdOxRPaBmS9FdQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+Matt Campbell gave a talk titled *AccessKit: reusable UI accessibility*, showing off AccessKit and its API and architecture.
+The talk also came with a cool demo, showcasing AccessKit's Android integration.
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/OI2TzJ6Sw10?si=FBRXVnlmasTLr_bF" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+### The Anniversary
+
+RustWeek also saw [the release of Rust 1.87](https://blog.rust-lang.org/2025/05/15/Rust-1.87.0/), which also fell exactly on the 10 year anniversary for Rust 1.0!
+
+(If you're like me and you're wondering how that works: Rust has one release every six weeks. `87 * 6 * 7 == 3654` and `3654 == 365 * 10 + 4`. Between 2015 and 2025, there have been four bissextile years: 2016, 2020, 2024, and... wait a minute. Well, turns out Rust 1.1 came out one day earlier than the six-weeks schedule implies, June 25 2015 instead of June 26 2015. There may have been other discrepancies, I haven't checked all the dates.)
```

Comment:
I think I'd probably just trim this entire section down to:
> RustWeek also saw [the release of Rust 1.87](...), which coincided with the 10 years of Rust party. Some members of Linebender were at the party, which was a great experience. See who you can spot [in the group photo](...).

This aside could make a really nice short personal blog post, or maybe a reddit post, though. Looking at this paragraph, it looks to me that Rust 1.0 was delayed by a day, rather than 1.1 was early; I'd presume that the cadence was always Thursdays, but 1.0 missed that for some reason).

Reply:
Yeah, I think that aside could work if it was shorter and we had more formatting options, but as it is it's kind of awkward. I'll make a blog post.